Source/WebCore/PAL/Configurations/PAL.xcconfig

```diff
@@ -66,6 +76,33 @@ AVIF_LIBTOOLFLAGS_YES = $(BUILT_PRODUCTS_DIR)/libavif.a;
 DAV1D_LIBTOOLFLAGS_YES = $(BUILT_PRODUCTS_DIR)/libdav1d.a;
 
 
-SWIFT_OBJC_INTERFACE_HEADER_NAME = PALSwift.h;
-SWIFT_INSTALL_OBJC_HEADER = YES;
+SWIFT_OBJC_INTERFACE_HEADER_NAME = PALSwift-Generated.h;
 SWIFT_VERSION = 5.0;
+SWIFT_OBJC_INTEROP_MODE = objcxx;
+
+OTHER_SWIFT_FLAGS = $(OTHER_SWIFT_FLAGS_$(WK_PLATFORM_NAME));
+OTHER_SWIFT_FLAGS_macosx = $(OTHER_SWIFT_FLAGS$(WK_MACOS_1400));
+OTHER_SWIFT_FLAGS_maccatalyst = $(OTHER_SWIFT_FLAGS$(WK_MACCATALYST_14));
+OTHER_SWIFT_FLAGS_iphoneos = $(OTHER_SWIFT_FLAGS$(WK_IOS_17));
+OTHER_SWIFT_FLAGS_iphonesimulator = $(OTHER_SWIFT_FLAGS$(WK_IOS_17));
+OTHER_SWIFT_FLAGS_appletvos = $(OTHER_SWIFT_FLAGS$(WK_TVOS_17));
+OTHER_SWIFT_FLAGS_appletvsimulator = $(OTHER_SWIFT_FLAGS$(WK_TVOS_17));
+OTHER_SWIFT_FLAGS_watchos = $(OTHER_SWIFT_FLAGS$(WK_WATCHOS_10));
+OTHER_SWIFT_FLAGS_watchsimulator = $(OTHER_SWIFT_FLAGS$(WK_WATCHOS_10));
+OTHER_SWIFT_FLAGS_visionos = -Xcc -std=c++20 -I $(SRCROOT)/pal -I $(SRCROOT)/PAL/pal;
```

Comment:
@emw-apple should this be `xros` ? or `visionos` ?

Reply:
xros. If you ever need to look this up, it's defined at the root of the SDK in `SDKSettings.json`.